cross/x264/Makefile

```diff
@@ -1,20 +1,52 @@
 PKG_NAME = x264
-PKG_VERS = 20141218-2245
+PKG_VERS = snapshot-20160517-2245-stable
```

Comment:
I didn't mind the old format: keep PKG_VERS the actual version, and adjust `PKG_DIST_NAME` and `PKG_DIR` accordingly.